ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -892,6 +892,40 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Ensure Home Folders Are Secure
+  platforms: macOS
+  platform: darwin
+  description: |
+    By default, macOS allows all valid users into the top level of every other user's home folder and restricts access to the Apple default folders within. Another user on the same system can see you have a "Documents" folder but cannot see inside it. This configuration does work for personal file sharing but can expose user files to standard accounts on the system.
+    The best parallel for Enterprise environments is that everyone who has a Dropbox account can see everything that is at the top level but can't see your pictures. Similarly with macOS, users can see into every new Directory that is created because of the default permissions.
+    Home folders should be restricted to access only by the user. Sharing should be used on dedicated servers or cloud instances that are managing access controls. Some environments may encounter problems if execute rights are removed as well as read and write. Either no access or execute only for group or others is acceptable.
+  resolution: |
+    Automated method:
+    Ask your system administrator to deploy a script that will go over all users and set the mode for all of them either like this:
+    /usr/bin/sudo /bin/chmod -R og-rwx /Users/<username>
+    Or like this if there is a need for excutable access:
+    /usr/bin/sudo /bin/chmod -R og-rw /Users/<username>
+  query: SELECT 1 FROM file WHERE 
+            path LIKE '/Users/%' 
+            AND 
+            path != '/Users/Shared/' 
+            AND 
+            (
+              mode = "0700" 
+              OR 
+              mode ="0701" 
+              OR 
+              mode ="0710" 
+              OR 
+              mode ="0711"
+            );
```

Comment:
We actually want to check if ALL directories have those permissions, AFAICS this query checks that there's at least one directory with the given permissions.

Something like this:
```sql
SELECT 1 WHERE NOT EXISTS (
  SELECT * FROM file
    WHERE path LIKE '/Users/%' AND path != '/Users/Shared/' 
    AND mode != "0700" AND mode != "0701" AND mode != "0710" AND mode != "0711"
);
```

Reply:
dohhh .. god catch.
Fixed.